src/main/java/com/dnsimple/data/DomainTransfer.java

```diff
@@ -51,6 +54,8 @@ public boolean hasWhoisPrivacy() {
     return whoisPrivacy;
   }
 
+  public String getStatusDescription() { return statusDescription; }
```

Comment:
```suggestion
  public String getStatusDescription() { 
    return statusDescription; 
  }
```

Reply:
I'm suggesting this to have it conforming to the rest of methods in the class. I'm guessing IntelliJ did that when automatic reformating the file. You can disable that by unchecking the `Simple blocks in one line` code style setting:
![image](https://user-images.githubusercontent.com/205913/82021860-ed0e9d80-968b-11ea-86da-178504e4b7ad.png)